src/platform/vita/cursor.c

```diff
@@ -11,17 +11,17 @@ vita2d_texture *current_cursor;
 
 static vita2d_texture *init_cursor(const cursor *c)
 {
-    vita2d_texture *tex = vita2d_create_empty_texture_format(32, 32, SCE_GXM_TEXTURE_FORMAT_U8U8U8U8_ARGB);
+    vita2d_texture *tex = vita2d_create_empty_texture_format(c->width, c->height, SCE_GXM_TEXTURE_FORMAT_U8U8U8U8_ARGB);
     color_t *cursor_buf = vita2d_texture_get_datap(tex);
```

Comment:
I'm not sure but sometimes textures need to be either divisable by 2 or a power of 2 in order to work, that's why I left them at 32x32 in size